src/NUnitFramework/framework/nunit.framework.csproj

```diff
@@ -6,6 +6,11 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net20|AnyCPU'">
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
```

Comment:
I had to do this to be able to debug. I can remove it if somebody can tell me why I'm unable to debug (break) using the portable pdb output, which is the default.

Reply:
I have a feeling this will be a problem. Are you running VS older than 2015 Update 2?

Reply:
VS2017 15.5.1

Reply:
That's perplexing. What was the startup project/executable at the time?

Reply:
Either the nunitlite test exe or the nunitlite runner - IIRC both had the problem.